src/physics/utils/phys_comp.F90

```diff
@@ -245,60 +202,76 @@ subroutine phys_run2(dtime_phys, cam_runtime_opts, phys_state, phys_tend,  &
            read_initialized_variables=use_init_variables)
 
       ! Initialize the physics time step
-      call cam_ccpp_physics_timestep_initial(phys_suite_name, dtime_phys,     &
-           errmsg, errcode)
+      call cam_ccpp_physics_timestep_initial(phys_suite_name)
```

Comment:
I'm surprised we no longer need to pass the dtime_phys variable

Reply:
I moved the dtime_phys variable to the registry so it is now "used" by the ccpp cap like ccpp_error_message and ccpp_error_code